### ISSUE TITLE
Current version does not appear to work - Sell button on vendor window does nothing.

### ISSUE THREAD

**Issue by ShrikeBTR** [Beast-Masters-addons/SellJunk#1]
This error comes up when bringing up the vendor:

```Message: Interface/AddOns/SellJunk/SellJunk.lua:93: attempt to call global 'GetContainerNumSlots' (a nil value)
Time: Mon Nov 21 14:31:40 2022
Count: 2
Stack: Interface/AddOns/SellJunk/SellJunk.lua:93: attempt to call global 'GetContainerNumSlots' (a nil value)
[string "=(tail call)"]: ?
[string "@Interface/AddOns/SellJunk/SellJunk.lua"]:93: in function `Sell'
[string "@Interface/AddOns/SellJunk/SellJunk.lua"]:71: in function `?'
[string "@Interface/AddOns/DataStore/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:114: in function <...ore/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:114>
[string "=[C]"]: ?
[string "@Interface/AddOns/DataStore/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:24: in function <...ore/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:20>
[string "@Interface/AddOns/DataStore/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:59: in function `Fire'
[string "@Interface/AddOns/DataStore/libs/AceEvent-3.0/AceEvent-3.0.lua"]:120: in function <.../AddOns/DataStore/libs/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals: <none>
```

This error comes up when clicking the "Sell Junk" button:
```Message: Interface/AddOns/SellJunk/SellJunk.lua:93: attempt to call global 'GetContainerNumSlots' (a nil value)
Time: Mon Nov 21 14:31:24 2022
Count: 1
Stack: Interface/AddOns/SellJunk/SellJunk.lua:93: attempt to call global 'GetContainerNumSlots' (a nil value)
[string "=[C]"]: in function `GetContainerNumSlots'
[string "@Interface/AddOns/SellJunk/SellJunk.lua"]:93: in function `Sell'
[string "@Interface/AddOns/SellJunk/SellJunk.lua"]:27: in function <Interface/AddOns/SellJunk/SellJunk.lua:27>

Locals: 
```

**Comment by datagutten** [comment]
Blizzard has replaced a lot of bag APIs, I will try to fix the addon.

**Comment by datagutten** [comment]
I have replaced the removed API calls and the latest alpha build should work on WoW retail, but it has broken Classic. I am working on a polyfill to make new API calls work on WoW Classic.
The latest build can be downloaded here: https://addons.wago.io/addons/selljunk/versions?stability=alpha

**Comment by datagutten** [comment]
I fixed WoW Classic and a version working on all game versions are released as v3.2 available on GitHub, Wago and WowInterface.
https://www.wowinterface.com/downloads/info26443-SellJunk.html
